### PR TITLE
Add Flickr30k task

### PR DESCRIPTION
**Pull request by vgonzalez88**
-Add Flickr30k task based on the [set compiled by UIUC](http://web.engr.illinois.edu/~bplumme2/Flickr30kEntities/)
-Only one dataset with no train/test/val delineations
-Uses an open-ended teacher similarly to VQA
-Tested using run_tests_short.sh

Need to make some stylistic changes to match the suggestions given in #773

### CODE REVIEW

File: parlai/tasks/flickr30k/agents.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+class DefaultDataset(FlickrDataset):
+    pass
+
+
+class OeTeacher(FixedDialogTeacher):
+    """Open-Ended teacher
+    """
+    def __init__(self, opt, shared=None):
+        super().__init__(opt)
```
Comment (alexholdenmiller) [inline parlai/tasks/flickr30k/agents.py]: `super().__init__(opt, shared)`

**Comment by alexholdenmiller** [comment]
thanks! can you also post a paste from `python examples/display_data.py -t flickr30k -im ascii`?

File: parlai/tasks/task_list.py [hunk 2]
```diff
@@ -383,4 +383,11 @@
         "description": "Dump of Wikipedia articles from 2/3/18",
         "notes": "Specify ':full' for the full articles to be returned, otherwise defaults to ':summary', which provides the first paragraphs. To put the article in the labels and the title in the text, specify ':key-value' at the end (for a title/content key-value association)"
     },
+    {
+        "id": "Flickr30k",
+        "display_name": "Flickr30k",
+        "task": "flickr30k",
+        "tags": ["All", "Visual"],
+        "description": "30k captioned images pulled from Flickr compiled by UIUC: http://web.engr.illinois.edu/~bplumme2/Flickr30kEntities/"
```
Comment (klshuster) [inline parlai/tasks/task_list.py]: Could you also put in the citations listed at the website here? Would be good to have those somewhere

**Comment by vgonzalez88** [comment]
Added a changelog to NEWS.md describing the tasks added, I included this task and COCO from #773 .

**Comment by klshuster** [comment]
Looks good to me!

**Pull request merged**